discovery/kubernetes/kubernetes.go

```diff
@@ -750,6 +770,22 @@ func (d *Discovery) newNodeInformer(ctx context.Context) cache.SharedInformer {
 	return d.mustNewSharedInformer(nlw, &apiv1.Node{}, resyncDisabled)
 }
 
+func (d *Discovery) newNamespaceInformer(ctx context.Context) cache.SharedInformer {
+	nlw := &cache.ListWatch{
+		ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
+			options.FieldSelector = d.selectors.namespace.field
+			options.LabelSelector = d.selectors.namespace.label
+			return d.client.CoreV1().Namespaces().List(ctx, options)
+		},
+		WatchFunc: func(options metav1.ListOptions) (watch.Interface, error) {
+			options.FieldSelector = d.selectors.node.field
+			options.LabelSelector = d.selectors.node.label
```

Comment:
I think you meant:
```suggestion
			options.FieldSelector = d.selectors.namespace.field
			options.LabelSelector = d.selectors.namespace.label
```
But because `namespace` isn't a role (see another comment), we can just remove those selectors from both list and watch